src/components/ProjectOverviewItemCard/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import strings from 'src/strings';
+import { useAppSelector } from 'src/redux/store';
+import { selectProjects } from 'src/redux/features/projects/projectsSelectors';
+import { useOrganization } from 'src/providers';
+import { isContributor } from 'src/utils/organization';
+import { AssignProjectRequestPayload } from 'src/services/ProjectsService';
+import ProjectAssignModal from 'src/components/ProjectAssignModal';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import { Link as LinkMUI, Theme, useTheme, Box } from '@mui/material';
+import { Icon } from '@terraware/web-components';
+import { makeStyles } from '@mui/styles';
+
+interface OverviewItemCardProjectProps<T extends { id: number; projectId?: number }> {
+  entity: T;
+  reloadData: () => void;
+  projectAssignPayloadCreator: () => AssignProjectRequestPayload;
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  addIcon: {
+    fill: theme.palette.TwClrIcnBrand,
+    height: '20px',
+    width: '20px',
+    marginRight: '5px',
+  },
+}));
+
+const ProjectOverviewItemCard = <T extends { id: number; projectId?: number }>({
+  entity,
+  reloadData,
+  projectAssignPayloadCreator,
+}: OverviewItemCardProjectProps<T>) => {
+  const theme = useTheme();
+  const { selectedOrganization } = useOrganization();
+  const userCanEdit = !isContributor(selectedOrganization);
+  const classes = useStyles();
+
+  const projects = useAppSelector(selectProjects);
+  const entityProject = projects?.find((project) => project.id === entity?.projectId);
+
+  const [isProjectAssignModalOpen, setIsProjectAssignModalOpen] = useState<boolean>(false);
+
+  const linkStyle = {
+    color: theme.palette.TwClrTxtBrand,
+    fontWeight: 500,
+    textDecoration: 'none',
+    cursor: 'pointer',
+    fontSize: '14px',
+  };
+
+  return (
+    <OverviewItemCard
+      isEditable={userCanEdit}
+      handleEdit={() => setIsProjectAssignModalOpen(true)}
+      title={strings.PROJECT}
+      contents={
+        <>
+          {entityProject?.name ?? (
+            <LinkMUI sx={linkStyle} onClick={() => setIsProjectAssignModalOpen(true)}>
```

Comment:
There's a Link component under common

Reply:
Yea... I was wondering about that. Just copied what was already in https://github.com/terraware/terraware-web/blob/ng/SW-4496-accession-project-edit/src/components/accession2/view/Accession2View.tsx#L4

I'll change it over though

Reply:
Ah, looks like it was that way because the `Link` in common doesn't take `sx` prop.

Reply:
Got it, that could have been older code that wasn't updated.

Reply:
I guess it isn't really a link, it is more of a button. Going to leave as is since we don't actually want to do any routing with this click

Reply:
The linkStyle should be standard across all links, that's one reason the Link component was created. Shouldn't need to use the sx prop in this scenario.

Reply:
> I guess it isn't really a link, it is more of a button. Going to leave as is since we don't actually want to do any routing with this click

Right, my mistake. Thought this was a text link. Perhaps we don't need the link here at all. Just a cursor pointer on the Box and an onClick. Ok fine to leave as-is.

Reply:
I switched it over, everything looks / works the same. Looks like the common `Link` component only does routing stuff if there's a `to` prop.

Reply:
> I switched it over, everything looks / works the same. Looks like the common `Link` component only does routing stuff if there's a `to` prop.

Thanks for the change. I was looking at the pencil icon, not 'add project' - ignore my last comment about not needing a link.